kubernetes/windows/setup.ps1

```diff
@@ -35,7 +35,7 @@ Write-Host ('Finished Installing Fluentbit')
 
 Write-Host ('Installing Telegraf');
 try {
-    $telegrafUri='https://dl.influxdata.com/telegraf/releases/telegraf-1.22.2_windows_amd64.zip'
+    $telegrafUri='https://dl.influxdata.com/telegraf/releases/telegraf-1.23.2_windows_amd64.zip'
```

Comment:
same here. please do test to ensure this version upgrade doesn't cause any regressions.